plugins/src/test/java/com/qubole/quark/plugins/jdbc/JdbcFactoryValidationTest.java

```diff
@@ -33,7 +33,7 @@ public void testGenericDb() throws QuarkException {
   public void testValidProps() throws QuarkException {
     Map<String, Object> props = new HashMap<>();
     props.put("type", "mysql");
-    props.put("url", "jdbc:quark:test");
+    props.put("url", "fatjdbc:quark:test");
```

Comment:
Also, connect string is "jdbc:quark:" for fatjdbc, and "jdbc:quark:thin:" for jdbc. I didn't make any changes in fatjdbc connect string.